### PR TITLE
feat: fake with multiple parameters

### PR DESCRIPTION
**Pull request by ST-DDT**
Adds support for method calls with multiple parameters in `fake()`

Example:

````ts
faker.helpers.fake('I rolled the dice and got: {{random.numeric(5, {"bannedDigits": "12345678"})}}')
````

Result:

`I rolled the dice and got: 6`

<details>
<summary>Docs Preview</summary>

grafik

</details>

### CODE REVIEW

**Comment by ST-DDT** [comment]
Blocked by #1458

File: src/modules/helpers/index.ts [hunk 1]
```diff
@@ -505,6 +509,7 @@ export class HelpersModule {
    * faker.helpers.fake('Good Morning {{name.firstName}}!') // 'Good Morning Estelle!'
    * faker.helpers.fake('You can call me at {{phone.number(!## ### #####!)}}.') // 'You can call me at 202 555 973722.'
    * faker.helpers.fake('I flipped the coin and got: {{helpers.arrayElement(["heads", "tails"])}}') // 'I flipped the coin and got: tails'
+   * faker.helpers.fake('I rolled the dice and got: {{random.numeric(5, {"bannedDigits": "12345678"})}}') // 'I rolled the dice and got: 6'
```
Comment (Shinigami92) [inline src/modules/helpers/index.ts]: You rolled a 6 but was banned and it was banned and it was not of length 5?
You are a really lucky guy :stuck_out_tongue_winking_eye:

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: src/modules/helpers/index.ts (hunk 1 above)
Comment (ST-DDT) [reply src/modules/helpers/index.ts]: Fixed

File: src/modules/helpers/index.ts [hunk 2]
```diff
@@ -568,22 +573,21 @@ export class HelpersModule {
...
     // Note: we experience a small performance hit here due to JSON.parse try / catch
     // If anyone actually needs to optimize this specific code path, please open a support issue on github
     try {
-      params = JSON.parse(parameters);
+      params = JSON.parse(`[${parameters}]`);
     } catch (err) {
       // since JSON.parse threw an error, assume parameters was actually a string
-      params = parameters;
+      if (parameters.length === 0) {
+        params = [];
```
Comment (Shinigami92) [inline src/modules/helpers/index.ts]: Codecov warns about this line is not covered :thinking:

Comment (ST-DDT) [reply src/modules/helpers/index.ts]: Fixed

**Review by Shinigami92 — DISMISSED** [review]

File: test/helpers.spec.ts [hunk 3]
```diff
@@ -488,31 +488,51 @@ describe('helpers', () => {
...
           const arr = ['one', 'two', 'three'];
-          const random = faker.helpers.fake(
+          const actual = faker.helpers.fake(
             '{{helpers.arrayElement(["one", "two", "three"])}}'
           );
-          expect(arr).toContain(random);
+          expect(arr).toContain(actual);
+        });
+
+        it.only('replaces a token with a random value for a method with an object parameter', () => {
```
Comment (Shinigami92) [inline test/helpers.spec.ts]:
```suggestion
        it('replaces a token with a random value for a method with an object parameter', () => {
```

**Review by Shinigami92 — DISMISSED** [review]

**Review by ejcheng — COMMENTED** [review]
Please replace all instances of `random.numeric` with `string.numeric` to fix tests

**Review by Shinigami92 — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**